### PR TITLE
LSR-919 Migration to swift 2.3

### PR DESCRIPTION
**Pull request by victorg1991**

### CODE REVIEW

**Comment by jmnavarro** [comment]
@victorg1991 conflicts in merge. Please, fix them together with @ahdezma

**Comment by victorg1991** [comment]
Done 👍

File: .travis.yml [hunk 1]
```diff
@@ -23,8 +23,6 @@ before_script:
         export LANG=en_US.UTF-8
         gem install cocoapods --version 0.39.0
         gem install xcpretty -N --no-ri --no-rdoc
-        brew update > /dev/null
-        brew outdated xctool || brew upgrade xctool
```
Comment (jmnavarro) [inline .travis.yml]: why remove this?

File: .travis.yml [hunk 2]
```diff
@@ -23,8 +23,6 @@ before_script:
         export LANG=en_US.UTF-8
         gem install cocoapods --version 0.39.0
```
Comment (jmnavarro) [inline .travis.yml]: @victorg1991 we don't need Cocoapods `0.39.0` anymore, right?

File: .travis.yml (hunk 1 above)
Comment (victorg1991) [inline .travis.yml]: We are not using xctool anymore, we currently use xcodebuild

File: .travis.yml (hunk 2 above)
Comment (victorg1991) [inline .travis.yml]: Yes, I'm making tests with travis in my repo

File: .travis.yml (hunk 1 above)
Comment (jmnavarro) [inline .travis.yml]: 👍

File: .travis.yml (hunk 2 above)
Comment (jmnavarro) [inline .travis.yml]: ok, push new commit when it's working with Cocoapods 1.0.x

**Comment by jmnavarro** [comment]
👍  pending to fix Cocoapod version in Travis

**Pull request merged**